### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by khickel** [ZOSOpenTools/ncursesport#11]
The LICENSE file at the top level says this is under the Apache License 2.0, the source repo for ncurses has it's open license that appears to be fairly open, but the LICENSE file in the patches subdirectory says GPL v3.0.  
Is the GPL v3.0 intended to cover any part of this, and if so, which part?
Thanks,
Kelly Hickel

**Comment by khickel** [comment]
@IgorTodorovskiIBM  as I've looked at more of the package repos, I've noticed other examples of this, should this be a broader discussion? It's  important to have clarity on the licensing.

**Comment by IgorTodorovskiIBM** [comment]
@khickel , thanks. We have some documentation around this in https://github.com/ZOSOpenTools/meta/blob/main/CONTRIBUTING.md#coding-guidelines . The top-level license represents the licensing for the zopen framework files. The LICENSE in the patches directory should represent the license of the tool being ported. 
We should add more details about the differences of both licenses. 

We also add this license to our metadata and will likely use it in the future to filter or view package licenses for each project

**Comment by khickel** [comment]
Thanks Igor, in that case when I find a zopen repo where the patches/LICENSE file does not match the license of the ported tool, should I open an issue against that zopen repo so that it can be corrected?

**Comment by IgorTodorovskiIBM** [comment]
> Thanks Igor, in that case when I find a zopen repo where the patches/LICENSE file does not match the license of the ported tool, should I open an issue against that zopen repo so that it can be corrected?

Yes @khickel , thank you. Better yet would be to open a PR with the correct LICENSE. But an issue is good too. Thanks!

**Comment by khickel** [comment]
Please correct the patches/LICENSE file to reference the MIT style license from the upstream repo rather than GPL v3.0.

**Comment by khickel** [comment]
> > Thanks Igor, in that case when I find a zopen repo where the patches/LICENSE file does not match the license of the ported tool, should I open an issue against that zopen repo so that it can be corrected?
> 
> Yes @khickel , thank you. Better yet would be to open a PR with the correct LICENSE. But an issue is good too. Thanks!

@IgorTodorovskiIBM The issue is that it's not easy to get approval to contribute changes to a repository that is has GPL v3 listed in the LICENSE file.
